vocab/vocab.csv

```diff
@@ -1077,71 +1077,71 @@ CP12CNSW-WFP01-00-WFPENG001,Coastal Pioneer MAB,Central Shallow Water Mooring,Wi
 CP12CNSW-WFP01-01-CTDPFP000,Coastal Pioneer MAB,Central Shallow Water Mooring,Wire-Following Profiler,CTD,Sea-Bird Electronics,SBE 37-SM,2,22
 CP12CNSW-WFP01-02-DOFSTP000,Coastal Pioneer MAB,Central Shallow Water Mooring,Wire-Following Profiler,Dissolved Oxygen,Aanderaa,Optode 4330F,2,22
 CP12CNSW-WFP01-03-FLORTP000,Coastal Pioneer MAB,Central Shallow Water Mooring,Wire-Following Profiler,3-Wavelength Fluorometer,WET Labs,ECO BBFL2-w,2,22
-CP12WESW,Coastal Pioneer MAB,Western Shallow Water Mooring,,,,,0,30
-CP12WESW-MFI01,Coastal Pioneer MAB,Western Shallow Water Mooring,Seafloor Multi-Function Node (MFN),,,,30,30
-CP12WESW-MFI01-02-ADCPTQ010,Coastal Pioneer MAB,Western Shallow Water Mooring,Seafloor Multi-Function Node (MFN),ADCP Velocity Profiler,Teledyne RDI,WHS600-200-waves [WHII],30,30
-CP12WESW-MFI01-02-CTDMOS031,Coastal Pioneer MAB,Western Shallow Water Mooring,Seafloor Multi-Function Node (MFN),CTD,Sea-Bird Electronics,SBE 37-IM,30,30
-CP12WESW-MFI01-02-WAVSSQ010,Coastal Pioneer MAB,Western Shallow Water Mooring,Seafloor Multi-Function Node (MFN),Surface Wave Spectra,Teledyne RDI,WHS600-200-waves [WHII],30,30
+CP12WESW,Coastal Pioneer MAB,Western Shallow Water Mooring,,,,,0,25
```

Comment:
Should this depth be 24 like the others?

Reply:
no the first line lists the full mooring depth, then the next few lines list the MFN depth, which we are considering 1 m up like we do for the SMs.